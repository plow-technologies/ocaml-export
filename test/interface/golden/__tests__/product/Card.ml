let () =
  AesonSpec.sampleGoldenAndServerSpec Card.decodeSuit Card.encodeSuit "suit" "http://localhost:8081/suit" "__tests__/golden/Suit.json";

  AesonSpec.sampleGoldenAndServerSpec Card.decodeCard Card.encodeCard "card" "http://localhost:8081/card" "__tests__/golden/Card.json";
