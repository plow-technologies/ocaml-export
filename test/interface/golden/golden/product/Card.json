{
    "seed": -7563482193342884554,
    "samples": [
        {
            "cardValue": 25,
            "cardSuit": "Spades"
        },
        {
            "cardValue": -18,
            "cardSuit": "Diamonds"
        },
        {
            "cardValue": 30,
            "cardSuit": "Hearts"
        },
        {
            "cardValue": 27,
            "cardSuit": "Spades"
        },
        {
            "cardValue": 1,
            "cardSuit": "Hearts"
        },
        {
            "cardValue": -22,
            "cardSuit": "Hearts"
        },
        {
            "cardValue": 1,
            "cardSuit": "Clubs"
        },
        {
            "cardValue": 16,
            "cardSuit": "Spades"
        },
        {
            "cardValue": -18,
            "cardSuit": "Spades"
        },
        {
            "cardValue": 4,
            "cardSuit": "Clubs"
        }
    ]
}